asyn/drvAsynSerial/drvAsynSerialPortWin32.c

```diff
@@ -224,27 +224,27 @@ static void monitorComEvents(void* arg)
 			}
 		}
 		getTimestamp(datetime, sizeof(datetime));
-		if (evtMask & EV_ERR)
+		if ((evtMask & EV_ERR) && (tty->commEventMask & EV_ERR))
```

Comment:
Could probably write these conditions as `evtMask & tty->commEventMask & EV_ERR` to avoid duplicating `EV_ERR` etc?

Reply:
Only for EV_ERR i am wondering if we should leave as  if (evtMask & EV_ERR)   as we should in general report errors